stable/dex-controller/templates/deployments.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: {{ template "dex-controller.fullname" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "dex-controller.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
```

Comment:
You could expose the replicas as a property, unless you think we don't support multiple replicas

Reply:
Done.